lib/internal/Magento/Framework/View/Layout/GeneratorPool.php

```diff
@@ -69,14 +72,17 @@ public function getGenerator($type)
     /**
      * Traverse through all generators and generate all scheduled elements
      *
-     * @param Reader\Context $readerContext
+     * @param Reader\Context    $readerContext
```

Comment:
No need to revert but please avoid such changes in general. Previous formatting was just fine.

Reply:
It was my IDE default formatting process. I missed it, I'm sorry.